jcl/src/java.management/share/classes/sun/management/DiagnosticCommandImpl.java

```diff
@@ -0,0 +1,23 @@
+/*[INCLUDE-IF Sidecar18-SE-OpenJ9&!Sidecar19-SE]*/
```

Comment:
Why is is this stubbed out in 8 only?

Reply:
The class was moved to com.sun.management.internal in Java 11+ and for Java 11+ it's already stubbed. 
https://github.com/eclipse/openj9/blob/master/jcl/src/jdk.management/share/classes/com/sun/management/internal/DiagnosticCommandImpl.java